Services/DataX.Config/DataX.Config/ConfigGeneration/Processor/S500_ResolveSparkJobParams.cs

```diff
@@ -26,10 +27,17 @@ public class ResolveSparkJobParams: ProcessorBase
         public const string TokenName_DatabricksToken = "guiSparkDatabricksToken";
         public const string TokenName_SparkJobDatabricksAutoScale = "guiSparkJobDatabricksAutoScale";
 
+        [ImportingConstructor]
+        public ResolveSparkJobParams(ConfigGenConfiguration conf)
```

Comment:
does the call to the ctor need to change?

Reply:
No caller to the constructor.as such.